### PR TITLE
Make /_astro have immutable cache headers

### PR DESCRIPTION
**Pull request by matthewp**
## Changes

- Adds immutable cache headers to `/_astro` files, which are all hashed.

## Testing

N/A

## Docs

N/A

### CODE REVIEW

File: .changeset/breezy-ants-rush.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/netlify': patch
```
Comment (matthewp) [inline .changeset/breezy-ants-rush.md:2]:
```suggestion
'@astrojs/netlify': minor
```

**Review by eduardoboucas — APPROVED** [review]

**Comment by ascorbic** [comment]
Oh no! Did you miss that I already did this in #326?

**Comment by ascorbic** [comment]
Oh no! We both did this! Mine is in #326.

**Review by alexanderniebuhr — COMMENTED** [review]
No preference which one to use, I like that @ascorbic's has tests

**Pull request closed without merging**